tests/benchmark/Dockerfile.in

```diff
@@ -1,6 +1,15 @@
-FROM ${FULL_BASE_IMAGE}
+FROM gcr.io/google-appengine/python:latest
 
-RUN hg clone -r 9923b81a1d34 https://hg.python.org/benchmarks /app/benchmarks
-WORKDIR /app/benchmarks
-RUN python perf.py -r -b default /usr/bin/python2.7 /usr/bin/python3.4
-RUN python perf.py -r -b default /usr/bin/python3.4 /opt/python3.5/bin/python3.5
+# Install performance
+RUN python2 -m pip install performance
+
+# Create virtual environment
+RUN python2 -m pip install -U virtualenv
+
+# Download ensurepip module which prevents the failure when benchmarking python3
```

Comment:
Oh I see, it's because debian drops ensurepip from it's python. Lame. Can you add a comment that says once we drop debian's 3.4 we can remove it?